test/expect/TestJit.test_script_fibb.expect

```diff
@@ -0,0 +1,35 @@
+graph(%lim : UNKNOWN_TYPE) {
+  %1 : UNKNOWN_TYPE = Constant[value={1}]()
+  %2 : UNKNOWN_TYPE = Constant[value={1}]()
+  %3 : UNKNOWN_TYPE = Constant[value={1}]()
+  %4 : UNKNOWN_TYPE = Constant[value={5}]()
+  %5 : UNKNOWN_TYPE = Constant[value={3}]()
+  %6 : UNKNOWN_TYPE = Constant[value={0}]()
+  %7 : UNKNOWN_TYPE = Constant[value={0}]()
+  %8 : UNKNOWN_TYPE = lt(%3, %lim)
+  %34 : UNKNOWN_TYPE, %35 : UNKNOWN_TYPE, %36 : UNKNOWN_TYPE, %37 : UNKNOWN_TYPE, %38 : UNKNOWN_TYPE = Loop(%7, %8, %1, %2, %6, %4, %3)
+    block0(%9 : UNKNOWN_TYPE, %10 : UNKNOWN_TYPE, %12 : UNKNOWN_TYPE, %25 : UNKNOWN_TYPE, %28 : UNKNOWN_TYPE) {
```

Comment:
It is annoying because `third` is not a real loop carried-dependency, except for the bit where it needs to be made available to the enclosing scope. But I guess to properly figure this out you'd need an actual fixpoint analysis.

Reply:
OK, I'm confused. Per the ONNX spec, `%9` is supposed to be the `iteration_num`, right? So why are you adding it to `%10`? It seems like the `iteration_num` and `cond` inputs are missing,

Reply:
Yeah this is confusing. I didn't add the iteration number input to the body block because it seemed redundant when user code is already creating that machinery. I was going to sync up with zach about it but forgot. Let me go ahead and add that input.

Reply:
Updated